### ISSUE TITLE
Consider adding checksum/signature validation to the script install process

### ISSUE THREAD

**Issue by raesene** [helm/helm-www#1090]
The [from script installation method](https://helm.sh/docs/intro/install/#from-script) involves downloading and running a script to install helm.

At the moment there's no mention in the documentation of checking the hash of the script or having a signature on the script to ensure it's validity. this could leave users of Helm at risk if an attacker was able to compromise the system hosting the script (as happened with [Codecov](https://about.codecov.io/security-update/)

It would be cool if the script was signed or a checksum was included in the documentation , with the instructions showing how to check that.

**Comment by bacongobbler** [comment]
Fair enough. That sounds like a reasonable request. Note that there are also flags in the script that can verify the release signatures for you as well (assuming you verified that you receive the correct script).

Feel like taking a crack at documenting this?

**Comment by manfred0191** [comment]
Since this issue is more than 2 years old, does anyone know if such a checksum and signature validation for helm is in place in between?

**Comment by bacongobbler** [comment]
There is both checksum and hash validation for helm-the-binary, which the script checks for. There is no checksum and hash validation for get-helm.sh.
